IndicoV2.IntegrationTests/DataSets/DataSetClientTests.cs

```diff
@@ -93,17 +104,21 @@ public async Task ProcessFiles_ShouldStartProcessing()
         public async Task ProcessCsv_ShouldStartProcessing()
         {
             // Arrange
-            var dataSet = await _dataHelper.DataSets().GetAny();
+            if (_dataSetId == 0)
```

Comment:
This logic would fit better in the DataSetHelper.cs class, if you pass the env variable inside the dependency injection portion of IndicoTestContainerBuilder w/something like(syntax might be a little diff, but this is the general idea):

`_container.RegisterType<DataHelper>(InjectionConstructor(datasetId);`

and have DataHelper injected/resolved in each test as it currently is.

otherwise, nice!